### ISSUE TITLE
Add tests for AstraUtils#getPackageName(TypeDeclaration)

### ISSUE THREAD

**Issue by RadikalJin** [alfasoftware/astra#59]
AstraUtils provides a utility for getting the package name for a type declaration. This isn't currently unit tested - coverage for this feature should be added to TestAstraUtils.

**Comment by shreelakshmijoshi** [comment]
Hi @RadikalJin !
I am a new open source contributor and would like to solve this issue, could you help me understand the issue better by providing some extra references ? 
Is this is issue open to contribute ?

**Comment by RadikalJin** [comment]
Hi @shreelakshmijoshi - thank you for your interest in Astra. Yes, this is open for contribution!

The idea here is to add test coverage for a utility function which gets the package name for a type declaration. 

```
package org.alfasoftware.astra.core.utils;

public class AstraUtils {
...
}
```
In this example, the type being declared is `AstraUtils`. The package name for that type is `org.alfasoftware.astra.core.utils`.


[Here's another test](https://github.com/alfasoftware/astra/blob/main/astra-core/src/test/java/org/alfasoftware/astra/core/utils/TestAstraUtils.java#L72) for AstraUtils, which gets the name for an anonymous class declaration. Hopefully we can write a similar test for our case. This will mean parsing a class (like the one written above - in the existing test this is `ExampleWithAnonymousClassDeclaration`, and using an ASTVisitor to get a reference to the TypeDeclaration, which we can pass to the utility for the test.

**Comment by arhaan023** [comment]
hey, is this still open for contribution? I'm new to contribution and would love this to be my first one.

**Comment by RadikalJin** [comment]
Hey @arhaan023, yes this is open for contribution, please feel free! Thanks for your interest in Astra.

**Comment by oussamabadr** [comment]
Hello

@arhaan023 are you still working on this issue?

I would like to open a pull request for this task...

**Comment by RadikalJin** [comment]
Closing now - this issue has been resolved by @oussamabadr under #77 and #79
